snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/SnomedRf2ImportRequest.java

```diff
@@ -260,7 +260,11 @@ ImportResponse doImport(final BranchContext context, final File rf2Archive, fina
 				updateLocales(context, codeSystemUri.getCodeSystem());
 			}
 			
-			return ImportResponse.success(visitedComponents.build(), reporter.getDefects());
+			if (Rf2ReleaseType.FULL == releaseType) {
```

Comment:
We could even use some sort of throw-away Set implementation that does not contain the added values so that it becomes empty forever. That we would only need to alter the builder variable above based on releaseType.